fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItemSettings.java

```diff
@@ -39,8 +39,7 @@
 	 * @return this builder
 	 */
 	public FabricItemSettings equipmentSlot(EquipmentSlotProvider equipmentSlotProvider) {
-		FabricItemInternals.computeExtraData(this).equipmentSlot(equipmentSlotProvider);
-		return this;
+		return custom(CustomItemSettingImpl.EQUIPMENT_SLOT_PROVIDER, equipmentSlotProvider);
```

Comment:
Change to `return this.custom(...` please otherwise change L51 to `return custom(...`